src/main/java/task/Tutorial.java

```diff
@@ -0,0 +1,32 @@
+package task;
+
+import java.time.LocalDateTime;
+import parser.DateTimeExtractor;
+
+/**
+ * This extension of the Event class will allow the student to add Tutorials to
+ * his schedule.
+ *
+ * @author Varun
+ */
+public class Tutorial extends Event {
+
+    public Tutorial(String description, LocalDateTime toDate, LocalDateTime atDate) {
```

Comment:
Same here. I hope you are very clear what is the specialized role of Tutorial class compare to Event class so that there is a need of having a child class.